os/drivers/iotdev/iotbus_irq.c

```diff
@@ -0,0 +1,297 @@
+/****************************************************************************
```

Comment:
Please do not make driver for just operation, driver should control just one device.
If you want get message or callback, you have to expand each of driver. (Generally, tinyara using mq for it.)